docs/pages/desktop-access/rbac.mdx

```diff
@@ -0,0 +1,137 @@
+---
+title: Desktop Access Access Controls
+description: Role-based access control (RBAC) for Teleport Desktop Access
+---
+
+# Desktop Access Role-Based Access Control
+
+Teleport's RBAC allows administrators to set up granular access policies for
+Windows desktops connected to Teleport.
+
+Teleport's "role" resource provides the following instruments for controlling
+desktop access:
+
+```yaml
+kind: role
+version: v4
+metadata:
+  name: developer
+spec:
+  options:
+    # Specify whether or not to record the user's desktop sessions.
+    # Desktop session recording is enabled if one or more of the user's
+    # roles has enabled recording. Defaults to true if unspecified.
+    # Desktop sessions will never be recorded if auth_service.session_recording
+    # is set to 'off' in teleport.yaml or if the cluster's session_recording_config
+    # resource has set 'mode: off'.
+    record_sessions:
+      desktop: true
+
+    # Specify whether clipboard sharing should be allowed with the
+    # remote desktop (requires a supported browser). Defaults to true
+    # if unspecified. If one or more of the user's roles has disabled
+    # the clipboard, then it will be disabled.
+    desktop_clipboard: true
+  allow:
+    # Label selectors for desktops this role has access to.
+    windows_desktop_labels:
+      environment: ["dev", "stage"]
+
+    # Windows user accounts this role can connect as.
+    windows_desktop_logins: ["Administrator", "{{internal.windows_logins}}"]
+```
+
+<Admonition type="warning" title="Active Directory Configuration">
+Teleport's RBAC system is not a replacement for proper Active Directory
+administration. Teleport-issued Windows certificates are valid for a small
+amount of time, but they do apply to the entire domain. Proper care should be
+taken to ensure that each Teleport user's roles reflect only the necesary
+Windows logins, and that these Windows users are properly secured.
+</Admonition>
+
+## Labeling
+
+Both `allow` and `deny` rules support `windows_desktop_labels` selectors. These
+selectors are matched against the labels set on the desktop. It is possible to
+use wildcards (`"*"`) to match all desktop labels.
+
+Windows desktops acquire labels in two ways:
+
+1. The `host_labels` rules defined in the Windows Desktop Service configuration
+2. Automatic `teleport.dev/` labels applied by Teleport (for desktops discovered
+   via LDAP only)
+
+For example, the following `host_labels` configuration would apply the
+`environment: dev` label to a Windows desktop named `test.dev.example.com`,
+and the `environment: prod` label to `desktop.prod.example.com`:
+
+```yaml
+  host_labels:
+  - match: '^.*\.dev\.example\.com$'
+    labels:
+      environment: dev
+  - match: '^.*\.prod\.example\.com$'
+    labels:
+      environment: prod
+```
+
```

Comment:
So far, I only see the "environment" key for labels. Does Desktop Access support arbitrary key/value pairs for labels in the `host_labels` section? Are there any labels a user would expect here?

Reply:
Totally arbitrary. `environment` is just an example.